docs/Azure_IoT_Integration_Guide/Azure_IoT_Central_integration/Configure_Azure_IoT_Central_integration.md

```diff
@@ -0,0 +1,35 @@
+# Connect integration
+
+To enable communication and data flow between the Azure IoT Central and Coiote DM platforms, you first need to integrate them using the dedicated extension module in Coiote DM. Follow the instruction below to learn how to do it.
+
+## Prerequisites:
+
+   - An active IoT Central with hub owner access permissions.
+   - A Coiote DM user account with permissions to use the integration extension.
```

Comment:
A Coiote DM user account with permissions to use the Hyperscaler Integration Center ?

Reply:
yes